src/rtcModule/webrtc.cpp

```diff
@@ -2107,14 +2126,7 @@ void Call::notifySessionConnected(Session& /*sess*/)
     if (mCallStartedSignalled)
         return;
 
-    if (mIsRingingOut && mChat.isGroup())
-    {
-        sendCallData(CallDataState::kCallDataSessionKeepRinging);
-    }
-    else
-    {
-        sendCallData(CallDataState::kCallDataSession);
-    }
+    sendCallData(CallDataState::kCallDataSession);
```

Comment:
If the call should still keep ringing, shouldn't we send the `CallDataState::kCallDataSessionKeepRinging`??